app/src/test/java/com/github/multimatum_team/multimatum/util/MockFirebaseDynamicLinks.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.multimatum_team.multimatum.util
+
+import android.content.Intent
+import com.google.android.gms.tasks.Tasks
+import com.google.firebase.dynamiclinks.FirebaseDynamicLinks
+import com.google.firebase.dynamiclinks.PendingDynamicLinkData
+import com.google.firebase.dynamiclinks.internal.DynamicLinkData
+import org.mockito.Mockito
+import org.mockito.Mockito.`when`
+import org.mockito.kotlin.any
+
+fun mockFirebaseDynamicLinks(): FirebaseDynamicLinks {
+    val dynamicLinks = Mockito.mock(FirebaseDynamicLinks::class.java)
+    `when`(dynamicLinks.getDynamicLink(any<Intent>())).then {
```

Comment:
No need to change this, but for future work if you don't know it already, I realized yesterday that there is a `whenever` method that does the same as `when` but is more handy to use as no ` are required.

Reply:
Thanks for the tip